### DISCUSSION TITLE
Meeting Agenda 3/28

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#356] [Meeting Agendas/Minutes]
1. Turn in weekly report
2. Review tasks from previous sprint
3. Discuss expo poster
4. Meet with Darian

**Comment by moreno-michael** [comment]
Meeting minutes 3/28

1. Weekly Report done and submission pending updated sprint screenshot

2. Task Review
- All tasks in progress

3. Discuss expo poster
- Label diagram with before and after for clarity

4. Meet with Darian
- Poster: Maybe change figure for load balancer to differentiate from application figure
- Poster: Maybe change the word "application" to server
- Poster: "Too much text"
- Poster: Simplify logic so it is easier for attendees to understand
- Poster: Explain difference between horizontal and vertical scaling with a diagram
- Task Review
